python/desc/imsim/sed_wrapper.py

```diff
@@ -38,10 +38,10 @@ def add_dust(self, sed_obj, Av, Rv, ext_type):
         """
         if (self.wavelen[ext_type] is None or
             not np.array_equal(sed_obj.wavelen, self.wavelen[ext_type])):
-            self.a[ext_type], self.b[ext_type] = sed_obj.setupCCMab()
+            self.a[ext_type], self.b[ext_type] = sed_obj.setupCCM_ab()
```

Comment:
Are these just a new interface with later versions of the interface code?  Or is it an actual different choice of functions?